src/components/views/audio_messages/Clock.tsx

```diff
@@ -18,7 +18,7 @@ import React, { HTMLProps } from "react";
 
 import { formatSeconds } from "../../../DateUtils";
 
-export interface IProps extends Pick<HTMLProps<HTMLSpanElement>, "aria-live"> {
+export interface IProps extends Pick<HTMLProps<HTMLSpanElement>, "aria-live" | "role"> {
```

Comment:
this goes against https://github.com/vector-im/element-meta/pull/216 where things should only have one export